WORKSPACE.bazel

```diff
@@ -0,0 +1,5 @@
+############################################
```

Comment:
Isn't this repo bzlmod-only now?

Reply:
Still need one non-bzlmod dep for rules_workflows. Will publish that to BCR in the future.